### PR TITLE
Update README wording

### PR DESCRIPTION
**Pull request by iNecas**
I was not sure what the current wording meant. Other variant I could
think of is `PR designed for that`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,5 +65,5 @@ Please comment on the enhancement issue to:
 - link to relevant issues in other repos
 
 Please do not comment on the enhancement issue to:
-- discuss a detail of the design, code or docs. Use a linked-to-issue or PR
-  design for that
+- discuss a detail of the design, code or docs. Use a linked-to-issue or design PR
```
Comment (ericavonb) [inline README.md]: Yeah, this sentence isn't very clear. It took me a bit to realize that enhancements are supposed to each have an issue tracking it beyond the enhancement proposal PR and doc. Maybe "PR for the enhancement proposal itself"?

**Comment by ericavonb** [comment]
It took me a few reads to understand this section as well. The README is not very clear about the process for creating an enhancement.

As I understand it, when you want to propose an enhancement, you create an issue (upstream has a template for it: [kubernetes/enhancements/ISSUE_TEMPLATE.md](https://github.com/kubernetes/enhancements/blob/master/ISSUE_TEMPLATE.md) to track the status of the enhancement across its lifecycle.

I don't see issues created for the PRs that have been opened. Should there be @derekwaynecarr ?

**Comment by iNecas** [comment]
I'm all for clarifying the process. If the tracking issues are not enforced, the readme should not even mention them

**Comment by wking** [comment]
Looking over open PRs, I saw this was still open.  I'm really not clear on the whole _When to Comment on an Enhancement Issue_ section.  Pedantically, we've only had [two issues][1] so far, compared with [115 PRs][2].  And things like [this][3] look like "comments on a detail of the design" to me.  Practically, it would be good to give some explicit best-practices examples to follow.  @derekwaynecarr, now that we have a few months under out belts in this repository, maybe link to some examples of the behavior you were trying to encourage?

[1]: https://github.com/openshift/enhancements/issues
[2]: https://github.com/openshift/enhancements/pulls
[3]: https://github.com/openshift/enhancements/pull/13#discussion_r328717104

**Comment by russellb** [comment]
I'm surprised this PR sat this long.  :)

As I read the change, it just changes "linked-to-issue or PR design ..." to "linked-to-issue or design PR ..." which seems like a pretty clear typo fix.  There's some other good points of discussion in here, but I think we should just merge the typo fix.

/approve
/lgtm

**Pull request merged**